src/cli_util.rs

```diff
@@ -1326,6 +1337,26 @@ pub fn handle_command_result(ui: &mut Ui, result: Result<(), CommandError>) -> i
             ui.write_error(&format!("Error: {}\n", message)).unwrap();
             2
         }
+        Err(CommandError::ClapCliError(inner)) => {
+            let clap_str = if ui.color() {
+                inner.render().ansi().to_string()
+            } else {
+                inner.render().to_string()
+            };
+
+            // Definitions for exit codes and streams come from
+            // https://github.com/clap-rs/clap/blob/master/src/error/mod.rs
```

Comment:
Ah, thanks for checking and not missing the two success cases!

Reply:
Credit goes to the test suite 😛  Specifically `cargo test --test test_global_opts  --test test_alias`